api/queries_repo.go

```diff
@@ -1245,7 +1261,112 @@ func ProjectNamesToPaths(client *Client, repo ghrepo.Interface, projectNames []s
 	return ProjectsToPaths(projects, projectsV2, projectNames)
 }
 
-// RelevantProjects retrieves set of Projects and ProjectsV2 relevant to given repository:
+// v1Projects retrieves set of RepoProjects relevant to given repository:
+// - Projects for repository
+// - Projects for repository organization, if it belongs to one
+func v1Projects(client *Client, repo ghrepo.Interface) ([]RepoProject, error) {
```

Comment:
The `v1Projects` and `v2Projects` functions are copy / pasted from `relevantProjects`. I cannot remove `relevantProjects` until I tackle `web mode` which has another code path to get there. It will go away. However, it now directly calls these two functions.